src/h5web/visualizations/heatmap/MappedHeatmapVis.tsx

```diff
@@ -13,7 +13,7 @@ interface Props {
   value: HDF5Value;
   dims: number[];
   title?: string;
-  axisMapping?: AxisMapping[];
+  axisMapping?: AxisMapping;
```

Comment:
Just moving the `[]` into `AxisMapping` itself for consistency with `DimensionMapping`.